resources/views/colorpicker.blade.php

```diff
@@ -16,19 +16,29 @@
                 window.addEventListener('filament-color-picker:init', () => {
                     this.picker = window.FilamentColorPicker.make($wire, {
                         parent: document.getElementById('filament-color-picker'),
-                        ...{{ \Illuminate\Support\Js::from($getPickerOptions()) }},
+                        ...@js($getPickerOptions()),
                     });
                 });
             },
         }"
     >
         <div
             id="filament-color-picker"
+            class="flex mt-1 rounded-md shadow-sm"
         >
+            @if ($getPreview())
+                <span class="items-center bg-white border border-r-0 border-gray-300 w-11 h-10inline-flex rounded-l-md sm:text-sm" :style="{ background: color }"></span>
```

Comment:
```suggestion
                <span class="items-center bg-white border border-r-0 border-gray-300 w-11 h-10 inline-flex rounded-l-md sm:text-sm" :style="{ background: color }"></span>
```

You're missing a space here.

Reply:
Oh nice catch! I'm pretty sure i've fixed that, maybe i've ctrl+z. 

`h-10` does not need to be there, it will assume the input height :)